datanommer.models/datanommer/models/__init__.py

```diff
@@ -41,6 +41,11 @@
 from sqlalchemy.sql import literal, select, exists
 from sqlalchemy.exc import IntegrityError
 
+# Fix missing unicode type in Python 3
+import sys
+if (sys.version_info >= (3,0)):
+    unicode = str
```

Comment:
Maybe a nicer option would be to use six.text_type? Unless you want to avoid introducing a new dependency on python-six.

Reply:
The new dependency would probably be fine.   This depends on `fedmsg` which already depends on `six`, so the dep is (really) already there.